File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.py
def solution(arr1, arr2):
    answer = [[0 for j in range(len(arr1[i]))] for i in range(len(arr1))]
    for r in range(len(arr1)):
        for c in range(len(arr1[r])):
            answer[r][c] = arr1[r][c] + arr2[r][c]
        #print(answer)
    return answer

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.java
class Solution {
    public int[][] solution(int[][] arr1, int[][] arr2) {
        int rowLength = arr1.length;
        int colLength = arr1[0].length;
        int[][] sum = new int[rowLength][colLength];
        for(int i=0; i<rowLength; i++){
            for(int j=0; j<colLength; j++){
                sum[i][j] = arr1[i][j] + arr2[i][j];
            }
        }
        return sum;
    }
}